stable_baselines3/common/policies.py

```diff
@@ -628,7 +628,10 @@ def evaluate_actions(self, obs: th.Tensor, actions: th.Tensor) -> Tuple[th.Tenso
         distribution = self._get_action_dist_from_latent(latent_pi)
         log_prob = distribution.log_prob(actions)
         values = self.value_net(latent_vf)
-        entropy = distribution.entropy()
+        try:
+            entropy = distribution.entropy()
+        except NotImplementedError:
+            entropy = None
```

Comment:
we could directly approximate the entropy here, no?

Reply:
I could be great, indeed.